UCLALibrary/californica
# chore: APPS-1575 Add calursus core to Solr8

To test this core: 
Setup a standalone solr8.11.1 instance locally
copy the solr8 files to new folder calursus under solr8/server/solr/

Run bin/solr start -f

open localhost:8983/solr in browser
You should see calursus core under cores drop down in GUI

To test it with Californica locally update solr_url in.even file to point to yourmachineipaddress:8983/solr/calursus
e.g SOLR_URL=http://192.168.0.229:8983/solr/calursus

and change the solr port in docker-compose to something different like 8990 as we wont be using the solr docker container for this testing

Then start californica docker compose, setup db and then open localhost:3000 to Test import a new collection.
